chore: update requestconfig doc (#1148)
SAP/cloud-sdk: docs-js/features/connectivity/generic-http-client.mdx

executeHttpRequest(destination, requestConfig, httpRequestOptions): Promise<HttpResponse>;
```

It takes the following parameters:

- The `destination` argument is of type `DestinationOrFetchOptions`
  - This can either be a <LinkToLatestJsApiDocs slug="interfaces/sap_cloud_sdk_connectivity.Destination" name="destination object" /> which was already fetched or an <LinkToLatestJsApiDocs slug="interfaces/sap_cloud_sdk_connectivity.DestinationFetchOptions" name="object containing a destination name and an optional JWT" />
- The `requestConfig` argument is of type <LinkToLatestJsApiDocs slug="types/sap_cloud_sdk_http_client.HttpRequestConfig" name="HttpRequestConfig" /> or <LinkToLatestJsApiDocs slug="types/sap_cloud_sdk_http_client.HttpRequestConfigWithOrigin" name="HttpRequestConfigWithOrigin" />.

>   - This parameter is optional

Reviewer: ```suggestion
  - This parameter is optional
  - The default value is a `GET` request.
```

- The `httpRequestOptions` argument is of type `HttpRequestOptions`
  - This parameter is optional

>   - Can be used to disable [automatic CSRF token fetching](#csrf-token-fetching)

Reviewer: ```suggestion
  - The default value is a request with [automatic CSRF token fetching](#csrf-token-fetching)
```

executeHttpRequest(destination, requestConfig, httpRequestOptions): Promise<HttpResponse>;
```

It takes the following parameters:

- The `destination` argument is of type `DestinationOrFetchOptions`
  - This can either be a <LinkToLatestJsApiDocs slug="interfaces/sap_cloud_sdk_connectivity.Destination" name="destination object" /> which was already fetched or an <LinkToLatestJsApiDocs slug="interfaces/sap_cloud_sdk_connectivity.DestinationFetchOptions" name="object containing a destination name and an optional JWT" />
- The `requestConfig` argument is of type <LinkToLatestJsApiDocs slug="types/sap_cloud_sdk_http_client.HttpRequestConfig" name="HttpRequestConfig" /> or <LinkToLatestJsApiDocs slug="types/sap_cloud_sdk_http_client.HttpRequestConfigWithOrigin" name="HttpRequestConfigWithOrigin" />.
  - This parameter is optional

>   - The default value is a `GET` request.

Author: ```suggestion
  - The default value is a `GET` request
```

- The `httpRequestOptions` argument is of type `HttpRequestOptions`
  - This parameter is optional
  - The default value is a request with [automatic CSRF token fetching](#csrf-token-fetching)

:::tip

This guide provides code snippets.
Fully executable example code can be found in the [SAP-samples/cloud-sdk-js repository on GitHub](https://github.com/SAP-samples/cloud-sdk-js/tree/main/samples/http-client-examples).

:::

In the most simple form, the function can be invoked like this:

```ts
const response = await executeHttpRequest({
  url: 'https://example.com'
});
```

> If the `requestConfig` value is not provided, the SDK treats it as a `GET` request.

Author: ```suggestion
In the example above, since the `requestConfig` value is not provided, the SDK treats it as a `GET` request.
```
